backend/utils/webhooks.py

```
@@ -1,69 +1,108 @@
 import asyncio
 import json
+from datetime import datetime
 from typing import List
 
 import requests
 import websockets
 
-from database.redis_db import get_user_webhook_db
+from database.redis_db import get_user_webhook_db, user_webhook_status_db, disable_user_webhook_db, \
+    enable_user_webhook_db, set_user_webhook_db
```

Ruff

`database.redis_db.set_user_webhook_db` imported but unused

Remove unused import: `database.redis_db.set_user_webhook_db`

(F401)